src/dymaptic.GeoBlazor.Core/Objects/TimeInterval.cs

```diff
@@ -0,0 +1,55 @@
+﻿using dymaptic.GeoBlazor.Core.Serialization;
+using System.Text.Json.Serialization;
+
+
+namespace dymaptic.GeoBlazor.Core.Objects;
+
+/// <summary>
+/// TimeInterval is a class that describes a length of time in one of ten temporal units such as seconds, days, or years.
+/// TimeInterval is referenced by many classes, such as TimeInfo, which is referenced by time-aware layers and the TimeSlider widget.
+/// <a target="_blank" href="https://developers.arcgis.com/javascript/latest/api-reference/esri-TimeInterval.html">
+///         ArcGIS
+///         JS API
+///     </a>
+/// Used by Feature Layer.
+/// </summary>
+public class TimeInterval
+{
+    /// <summary>
+    /// Temporal units.
```

Comment:
Looks good, I saw that there was also a timeExtent class.  It just has 2 properties, just like this: a start and end property.  I see how the interval class is essential to the slider, I guess the other one is more for "displays"?
https://developers.arcgis.com/javascript/latest/api-reference/esri-TimeExtent.html